### ISSUE TITLE
Plot limits are not respected

### ISSUE THREAD

**Issue by jzkelter** [NetLogo/Galapagos#402]
Sometimes with plots in NetLogo web, the y-limits will automatically be resized when I don't want them to.

**Comment by TheBizzle** [comment]
You're right, I'm pretty sure I've run into this before.  You don't happen to have an example of a model (or sequence of actions in NLW) that causes this to happen, do you?
